### PR TITLE
feat/Add leader support

### PR DESCRIPTION
**Pull request by santoshbagadi**

### CODE REVIEW

**Comment by santoshbagadi** [comment]
Changes made.

File: Objective-C Demo App/iOS-SDK-Demo/ALDemoRootViewController.m [hunk 1]
```diff
@@ -93,15 +95,15 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
 }
 
 - (void)tableView:(UITableView *)tableView willDisplayCell:(UITableViewCell *)cell forRowAtIndexPath:(NSIndexPath *)indexPath {
```
Comment (thomasmso) [inline Objective-C Demo App/iOS-SDK-Demo/ALDemoRootViewController.m]: Could you also format this `{` to be on the next line?

File: Objective-C Demo App/iOS-SDK-Demo/ALDemoRootViewController.m [hunk 2]
```diff
@@ -92,6 +94,22 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
...
+    }
+}
+
+- (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPath *)indexPath
+{
+    if ( UIDevice.currentDevice.userInterfaceIdiom == UIUserInterfaceIdiomPhone && indexPath.section == 0 && indexPath.row == kRowIndexToHideForPhone )
+    {
+        return 0;
+    }
+    return [super tableView:tableView heightForRowAtIndexPath:indexPath];
```
Comment (thomasmso) [inline Objective-C Demo App/iOS-SDK-Demo/ALDemoRootViewController.m]: Also, AppLovin coding style has a ` ` char after the `:`, before the param

**Comment by santoshbagadi** [comment]
Fixed style errors.

**Pull request merged**